demo-app/src/components/RenderFromEndpoint.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import {
+  applyConfiguration,
+  parseYaml,
+  type COMPONENTS,
+} from '@open-truss/open-truss'
+
+export default function RenderFromEndpoint({
+  components = {},
+  url,
+}: {
+  components?: COMPONENTS
+  url: string
+}): JSX.Element {
+  // TODO: Use UQI's REST client once that exists?
```

Comment:
Ooh yeah, I like it 👍🏻 I can work towards this.

Reply:
Yes! Happy to collab too